src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -150,3 +219,20 @@ export function findPathRelativeToContent(notebookFolder: string, contentPath: U
 	}
 	return '';
 }
+
+export function escapeAngleBrackets(textContent: string) {
+	let text: string = textContent;
+	if (text.includes('<u>') || text.includes('<mark>') || text.includes('style')) {
```

Comment:
Are there any issues here if the tags contain "style" in their enclosed text, rather than as just an attribute?

Reply:
Also, what about a really long html string that contains <mark> or <u> tags in some places, but has text without those tags in others? Would that still be getting escaped properly? Like if you have underlined text before a list, for example.

Reply:
Discussed offline. The turndown replacement function deals with individual elements, so we don't need to worry about weird layering like this.

Reply:
Yup, all those scenarios should escape angle brackets properly since turndown breaks up HTML elements and the rules that are I introduced to ensure each Markdown Toolbar actions escapes the non-HTML tag, regardless if they contain multiple HTML tags.

Reply:
Although the "style" text comment is still valid.

Reply:
The edge case seems to be only when someone would want escape \<style\> properly as text. I'll make sure to update the condition specifically for "style".